baseui/templates/baseui/macros.html

```diff
@@ -0,0 +1,50 @@
+{# macros to be available to all templates across arxiv #}
```

Comment:
I like this! As they grow more numerous, would you continue to keep them in one file? Or split them out into separate files?

Reply:
Single vs multiple files: I suppose that depends on how many macros we end up needing. Ideally, we'd have some kind of list of components at this level in advance of the template creation...but we gotta work with what we've got.
I was imagining that we'd have separate macro files per section if the functions are section-specific, but I couldn't think of any and decided that doing it as one file for now was pragmatic.

If the macros start to emerge into groupings, separate files might be easier to manage.